pkg/kyverno/test/test_command.go

```diff
@@ -397,9 +397,9 @@ func testCommandExecute(dirPath []string, fileName string, gitBranch string, tes
 				continue
 			}
 
-			if strings.Contains(file.Name(), fileName) || strings.Contains(file.Name(), "test.yaml") {
+			if file.Name() == fileName {
```

Comment:
This must also include the test.yaml condition. That will allow us to cherry pick it back to 1.6.x and then we can submit a subsequent PR to main to remove test.yaml support.